<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="SettingsBox">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>440</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QSpinBox" name="WidthBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>48</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="HeigthLabel">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>55</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLabel" name="WidthLabel">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>86</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="HeigthBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>48</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="EnablePlayer">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>110</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable player</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StartGame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start game</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="AddSimpleBots">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>410</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add simple bots</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SimpleBotsCount">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>410</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddPlayer">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>380</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add player</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMunu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="actionRestart"/>
   </widget>
   <addaction name="menuMunu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRestart">
   <property name="text">
    <string>&amp;Restart</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
